packages/manager/apps/nutanix/package.json

```diff
@@ -19,6 +19,7 @@
     "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-nutanix-app' --include-dependencies -- npm run dev:watch --if-present"
   },
   "dependencies": {
+    "@ovh-ux/manager-bm-server-components": "^0.0.0 || ^1.0.0",
```

Comment:
Seems like some dependencies are missing here, `@ovh-ux/ng-at-internert`, `@ovh-ux/ng-ovh-feature-flipping` required by this module